ko/sql/schema/server_stmt.rst

```diff
@@ -0,0 +1,378 @@
+    RENAME SERVER dev1.srv1 AS srv3;
+
+..
+
+   위 예제가 *dba* 계정에서 수행 되었다면 *srv1* 서버의 소유자가 *cub* 였다면 변경된 *srv2* 의 소유자도 *cub* 이다.
+   또한, *srv3* 서버의 소유자는 *dev1* 으로 계속 유지 된다. 
+
+
+DROP SERVER
+===========
+
+**DROP SERVER** 구문을 이용하여 기존의 서버를 제거할 수 있다. **IF EXISTS** 절을 함께 사용하면 해당 서버가 존재하지 않더라도 에러가 발생하지 않는다.
+
+::
+
+    DROP SERVER [IF EXISTS] [owner_name.] server_name  ;
+    
+*   *owner_name*: 제거할 서버의 소유자 이름을 지정한다.
+*   *server_name*: 제거할 서버의 이름을 지정한다.
+
+
+.. code-block:: sql
+
+    DROP SERVER srv1;
+    DROP SERVER cub.srv1;
+    DROP SERVER IF EXISTS srv2;
+    
+    
+
+.. Warning::
+
+    DROP USER 구문으로 사용자를 삭제 할 때 해당 사용자가 소유하고 있는 서버가 있다면 에러 처리되고 사용자는 삭제되지 않는다.
+    먼저 사용자가 소유하고 있는 서버를 DROP SERVER 구문으로 제거한 후에 사용자 계정을 삭제해야 한다.
+        
+::
+   
+    -- When the current account is dba   
+    csql> create user cub;
+    Execute OK. (0.000371 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql> create server cub.tsrv (HOST='localhost', PORT=3300, DBNAME=demdb, USER=dev1);
+    Execute OK. (0.000761 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql> drop user cub;
+
+    In the command from line 1,
+
+    ERROR: Cannot drop the user who owns database objects(class/trigger/serial etc).
+
+    0 command(s) successfully processed.
+    csql> drop server cub.tsrv;
+    Execute OK. (0.000761 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql> drop user cub;
+    Execute OK. (0.001650 sec) Committed.
+
+    1 command(s) successfully processed.
+    csql>
+
+
+..
+
+위 예시를 보면 *cub* 계정 소유의 *tsrv* 서버를 생성 한 상태에서 drop user *cub* 구문이 실패하고 있음을 볼 수 있다.
+이후 *cub.tsrv* 서버를 제거 한 후에는 정상적으로 *cub* 게정을 삭제할 수 있었음을 볼 수 있다.     
+
+
+
+.. _alter-server:
+
+ALTER SERVER
+=============
+
+**ALTER** 구문을 이용하여 서버의 정보를 변경할 수 있다. 대상 서버의 소유자를 변경하거나, HOST, PORT, DBNAME, USER, PASSWOED, PROPERTIES, COMMENT에 대한 정보를 갱신 할 수 있다.  
+
+::
+
+    ALTER SERVER <dblink_server_name> <alter_server_list> ;
+     
+        <dblink_server_name> ::=  [owner_name.] server_name 
+        
+        <alter_server_list> ::=
+                <alter_server_list>, <alter_server_item>
+                | <alter_server_item>						   
+        <alter_server_item> ::= 
+                OWNER TO owner_name
+                | CHANGE <connect_item>
+        <connect_item> ::= 
+                HOST = host_string
+                | PORT = port_number 
+                | DBNAME = db_name 
+                | USER = user_name
+                | PASSWORD = [password_string]
+                | PROPERTIES = [properties_string] 
+                | COMMENT = [server_comment_string]
+                
+
+.. note::
+
+    서버의 소유자 또는 소유 그룹의 멤버만 정보를 변경 할 수 있다. 
+    특별히 **DBA** 또는 **DBA** 의 멤버는 모든 서버 정보를 변경할 수 있다.
+
+.. warning::
+
+    HOST, PORT, DBNAME, USER에 대한 값을 제거하는 갱신을 할 수는 없다.
+
+
+.. _owner_to:
+
+OWNER TO 절
+----------------
+
+**OWNER TO** 절을 사용하여 서버의 소유자를 변경할 수 있다. 
+
+::
+
+    ALTER SERVER  [owner_name.] server_name  OWNER TO new_owner_name ;

```

Comment:
There are two spaces, is that intentional?

Reply:
It was not intended. I'll fix it.